tools/skel-pack/package.js

```diff
@@ -1,8 +1,13 @@
 Package.describe({
   name: '~name~',
-  summary: ' /* Fill me in! */ ',
-  version: '1.0.0',
-  git: ' /* Fill me in! */ '
+  // Brief, one-line summary of the package.
+  summary: '',
+  version: '0.0.1',
+  // By default, Meteor will default to using README.md for documentation. Use
```

Comment:
Hmm. I see your point. However:
 - In the past, when we didn't put optional keys in package.js, we got multiple pull requests for it. I think it is probably a sign that we should consider erring on the side of putting it in, or it will look like we forgot it.
 - We are introducing a new thing, with the implicit README.md usage and asking for a *second* file (we used to only care about package.js!). People aren't used to this yet. Having it be there, and implicit, seems less confusing.
- There are some messages asking you to change the field, it is easier to do (for a new user) if the field is already there as an example. I know that we don't want them to do that, but I think that it might cause a lot of frustration to see confusing instructions.

We might be able to remove this in a few months when people are more used to it, but I am kind of wary of *not* putting it in right now.